sabnzbd/nzbstuff.py

```diff
@@ -1696,10 +1699,12 @@ def has_duplicates(self):
                 logging.debug('Dupe checking NZB against backup: filename=%s, result=%s', self.filename, res)
         # dupe check off nzb filename
         if not res and no_series_dupes:
-            series, season, episode, dummy = sabnzbd.newsunpack.analyse_show(self.final_name)
-            res = history_db.have_episode(series, season, episode)
-            series = res
```

Comment:
notice this fixes the 2015 bug that I reported -- https://github.com/sabnzbd/sabnzbd/commit/74e5f192525d55f1f997c82e514bbd5313092814#commitcomment-13708489